packages/sdks/src/components/render-block/render-component.lite.tsx

```diff
@@ -3,12 +3,14 @@ import BlockStyles from './block-styles.lite';
 import RenderBlock from './render-block.lite';
 import { For, Show, useMetadata } from '@builder.io/mitosis';
 import type { BuilderContextInterface } from '../../context/types.js';
+import type { Breakpoints } from '../../types/builder-content.js';
 
 export interface RenderComponentProps {
   componentRef: any;
   componentOptions: any;
   blockChildren: BuilderBlock[];
   context: BuilderContextInterface;
+  customBreakpoints?: Breakpoints;
```

Comment:
I'm confused about this prop. It's not being used anywhere in this component. Can it be removed?